include/seqan3/utility/views/all.hpp

```diff
@@ -88,20 +88,23 @@
  *
  * ### View properties
  *
- * There are three view properties that are documented for a view, **only if that view fulfills them:**
+ * There are three view properties that are documented for a view, **only if that view fulfils them:**
  *
  * **Source-only views:** Most views operate on an underlying range and return a (modified) range, i.e. they can be placed
  * at the beginning, middle or end of a "pipe" of view operations. However, some views are limited to being at
- * the front ("source"), e.g. `std::views::single`, `ranges::view::concat` and `std::views::ints`. These views
- * are marked as "source-only" and have no `urng_t` column in the second table.
+ * the front ("source"), e.g. `std::views::empty`, `std::view::single` and `std::views::iota`. These views
+ * are marked as "source-only" and have no `urng_t` column in the second table. The C++ standard calls these [Range
+ * factories](https://eel.is/c++draft/range.factories), which are utilities to create a view.
  *
  * **Sink-only views:** The opposite of a *source-only view*. It can only be placed at the end of a pipe, i.e.
  * it operates on views, but does not actually return a range (has no `rrng_t` column in the second table).
+ * The C++ standard doesn't have this concept yet.
```

Comment:
To make this more "timeless" maybe:
```suggestion
 * The C++20 standard doesn't have this concept.
```